Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,39 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    describe "when f.plist is not null" do
```

Comment:
Since I'm not using "when" in my code now, should I prefer changing some blocks to use `context` and "when" instead of `it`, @reitermarkus ?